### PR TITLE
Improve documentation of destination_conf parameter

### PR DESCRIPTION
**Pull request by victor-perov**
### Summary

This commit sheds more light on destination_conf attribute of logpush job configuration by adding details on the format it has and stresses more details on accepted schemes such as r2, sumo, etc.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [x] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Comment by victor-perov** [comment]
@soheiokamoto @angelampcosta please review when you get a chance. This change was motivated by a set of customer support tickets.

**Comment by victor-perov** [comment]
i will ignore those two ai suggestions on this occasion.

File: src/content/docs/logs/get-started/api-configuration.mdx [hunk 1]
```diff
@@ -93,16 +93,40 @@ You can specify your cloud service provider destination via the required **desti
...
+* `s3`,
+* `gs`,
+* `cos`,
+* `sumo`,
+* `azure`,
+* `splunk`,
+* `datadog`,
+* `https`,
+* `r2`,
+* `kinesis`.
```
Comment (cmackenzie1) [inline src/content/docs/logs/get-started/api-configuration.mdx]: Can we omit `kinesis` one for now? It is not GA yet.

Comment (victor-perov) [reply src/content/docs/logs/get-started/api-configuration.mdx]: Thanks, Cole! Updated

**Review by cmackenzie1 — APPROVED** [review]

File: src/content/docs/logs/get-started/api-configuration.mdx [hunk 2]
```diff
@@ -93,16 +93,39 @@ You can specify your cloud service provider destination via the required **desti
...
+* **Splunk** (scheme `splunk`): Splunk endpoint URL + Splunk channel ID + insecure-skip-verify flag + Splunk sourcetype + Splunk authorization token; for example: `splunk://<SPLUNK_ENDPOINT_URL>?channel=<SPLUNK_CHANNEL_ID>&insecure-skip-verify=<INSECURE_SKIP_VERIFY>&sourcetype=<SOURCE_TYPE>&header_Authorization=<SPLUNK_AUTH_TOKEN>`
+* **Sumo Logic** (scheme `sumo`): HTTP source address URL with `https` replaced by `sumo`; for example: `sumo://<SUMO_ENDPOINT_URL>/receiver/v1/http/<UNIQUE_HTTP_COLLECTOR_CODE>`
+
+For **R2**, **S3**, **Google Cloud Storage**, and **Azure**, logs can be separated into daily subdirectories by using the special string `{DATE}` in the URL path; for example: `s3://mybucket/logs/{DATE}?region=us-east-1&sse=AES256` or `azure://myblobcontainer/logs/{DATE}?[QueryString]`. It will be substituted with the date in `YYYYMMDD` format, like `20180523`.
```
Comment (angelampcosta) [inline src/content/docs/logs/get-started/api-configuration.mdx]:
```suggestion
For **R2**, **S3**, **Google Cloud Storage**, and **Azure**, logs can be separated into daily subdirectories by using the special string `{DATE}` in the URL path; for example: `s3://mybucket/logs/{DATE}?region=us-east-1&sse=AES256` or `azure://myblobcontainer/logs/{DATE}?[QueryString]`. The parameter will be substituted with the date in `YYYYMMDD` format, like `20180523`.
```

**Review by angelampcosta — APPROVED** [review]

**Pull request merged**